src/components/Liquid.vue

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
Is it necessary to keep track of liquids? What are some reasons why people might want to use this?